var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -34,11 +34,13 @@ class Graphviz(AutotoolsPackage):
 
     version('2.38.0', '5b6a829b2ac94efcd5fa3c223ed6d3ae')
 
+    # Swig can be enabled on it's own, though it doesn't do much.  It
+    # will also be enabled if any of the languages are enabled.
+    variant('swig', default=False,
+            description='Enable swig language binding tools')
```

Comment:
hmmm, `self.untested_bindings` and `self.tested_bindings` seem to work inside the method.